src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java

```diff
@@ -0,0 +1,69 @@
+package io.github.bobocodebreskul.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.github.bobocodebreskul.context.annotations.Get;
+import io.github.bobocodebreskul.context.registry.BringContainer;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DispatcherServlet extends HttpServlet {
+
+  private final BringContainer container;
+
+  private final Map<String, Object> pathToController;
+
+  public DispatcherServlet(BringContainer container, Map<String, Object> pathToController) {
+    this.container = container;
+    this.pathToController = pathToController;
+  }
+
+  private static Optional<Method> getGetMethod(Object bean) {
+    return Arrays.stream(bean.getClass().getMethods())
+        .filter(m -> m.getAnnotation(Get.class) != null).findFirst();
+  }
+
+  @SneakyThrows
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+      throws ServletException, IOException {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
I assume that we will use ObjectMapper for doPost, doPut and doDelete methods either, so we can make it as a global variable.